scripts/world_bank/wdi/worldbank.py

```diff
@@ -124,8 +124,9 @@ def read_worldbank(iso3166alpha3, fetchFromSource):
                 if df is None:
                     df = pd.DataFrame(columns=cols)
                 else:
-                    df = df.append(pd.DataFrame([cols], columns=df.columns),
-                                   ignore_index=True)
+                    # df = df.append(pd.DataFrame([cols], columns=df.columns),
```

Comment:
Add a comment?

Reply:
removed df.append() calls as the Api is deprecated

Reply:
Should we delete instead of comment the old line?

Reply:
yes, deleted now. was commented for testing.